app/pages/Auction/BidActionPanel/Reveal.js

```diff
@@ -143,12 +145,17 @@ class Reveal extends Component {
       text = 'Reveal Pending...';
     }
 
+    if (needsRepair) {
+      text = 'Repair unknown bids';
```

Comment:
Please capitalize: `Repair Unknown Bids`.

Reply:
fixed in 1521a6672fad0dc28b9b0e3da014fb33ccf7b12e